usr.sbin/freebsd-update/freebsd-update.sh

```diff
@@ -1215,6 +1215,16 @@ workdir_init () {
 	touch tINDEX.present
 }
 
+# Fetch compare upstream tag (tag.new) to tag on disk
+fetch_compare_new_tag_to_current_tag () {
+	if [ -f tag.new -a -f tag ] && cmp -s tag.new tag; then
+		echo "Fetched metadata tag match."
```

Comment:
Remember that it is still leaving a `tag.new` file. Maybe do what it's being done in:
https://github.com/freebsd/freebsd-src/blob/bac1212275d1c583d6e6049b361b12ce8228c028/usr.sbin/freebsd-update/freebsd-update.sh#L1541

Reply:
I've opted for "rm tag.new", my reason for this is that I think the file 'tag' should still only be a result of the function that builds it.